### DISCUSSION TITLE
Need help

### DISCUSSION THREAD

**Discussion by Dave48529** [community/community#144584] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

My wife started cheating with someone who keeps putting some sort of rootkit malware on my phone to control it. I have been able to see some of the code comes from GitHub but not much else can someone please tell me how to delete the code or how access the hidden files/pics etc. they put all sorts of shit on my phone to mentally torture me basically flaunting they can’t be caught! Someone please help me. This asshole put mini cams in my bedroom and bathroom were my kids change and shower and I need to put a stop to it. PLEASE HELP

**Comment by Hyphonical** [comment]
I would suggest reporting this to your local police, if the guy is filming your kids. This is not something Github can handle

**Reply by Dave48529** [reply]
I tried taking my phone into police to get them to bypass the malware on it. Basically because they couldn't see any financial crimes or the proof of mini hidden cams they couldn't go through. He had my wife install some sort of rootkit malware on my phone so they can control it from there devices. I need to bypass this code so I can get to the files showing he has hidden cams. They are so arrogant they send me videos with overlays on them that are of fucked up shit but I can't open them. I need to know how to bypass this source code to unlock all the files they put on my damn phone

**Reply by Hyphonical** [reply]
So this guy instructed your wife to put malware on your phone so the guy can control your device, and he put a lock on the malware so you cant see what videos the mini cams have made? Well, if your phone is connected to the camera's and that device solely controls it, i would suggest you disable the Wi-Fi and Bluetooth on that device, and give it to a repair shop or any phone expert that can safely remove this malware, and you should definitely find those camera's. On the other hand, if he put a new password on your phone so you cant login, i would still do the same, send it to an expert and tell them to make a copy of your files and reset your phone. Rootkit malware isn't a joke, i assume its an android, so bypassing the system would probably be possible somehow.

**Comment by prothesbarai** [comment]
Response #2

**Reply by Hyphonical** [reply]
What is this man, this is not helpful

**Comment by Akash1134** [comment]
Hi @Dave48529 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
